stacks/container-apps/exchange.prx.org.yml

```diff
@@ -59,6 +63,7 @@ Parameters:
     Type: String
   CreateWorker:
```

Comment:
Seems like you could just remove the `CreateWorker` param and all the `CreateWorkerResources` conditionals.  Since we always create the worker here.